tests/Helper/InternalHelperTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace PHPSu\Tests\Helper;
+
+use PHPSu\Exceptions\CommandExecutionException;
+use PHPSu\Exceptions\EnvironmentException;
+use PHPSu\Helper\InternalHelper;
+use PHPSu\Tools\EnvironmentUtility;
+use PHPUnit\Framework\TestCase;
+
+class InternalHelperTest extends TestCase
+{
+
+    private const GIT_PATH = __DIR__ . '/../../.git';
+
+    public function testGetPhpsuVersionFromVendor(): void
+    {
+        $result = $this->callPrivateMethod('getPhpSuVersionFromVendor');
+        // PHPSU_VENDOR_INSTALLATION === false
+        $this->assertEmpty($result, 'Asserting phpsu-vendor version to be empty due to test context');
+    }
+
+    public function testGetPhpSuVersionFromGitFolder(): void
+    {
+        $this->assertFileExists(self::GIT_PATH . '/HEAD');
```

Comment:
Can we use a small fixture file for this test?
To test if it returns the right version